src/formats/GRO.cpp

```diff
@@ -61,14 +61,17 @@ static void check_values_size(const Vector3D& values, unsigned width, const std:
 void GROFormat::read_next(Frame& frame) {
     residues_.clear();
 
+    // GRO comment line is used as frame name
+    auto name = trim(file_.readline());
+    if (!name.empty()) {
+        frame.set("name", name.to_string());
+    }
+
```

Comment:
Just for my understanding: Why do you set 'name' in formats like GRO or MOL2 only if it's a non-empty string, but in formats like SDF the comment is not checked?

Reply:
Because I forgot to add the check in SDF!